### PR TITLE
Update bot.py

### PR DESCRIPTION
**Pull request by FarmTon**
Fixed timer stuck on 0:01

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -126,13 +121,14 @@ def main(self):
             _end = int(time.time())
             _tot = _end - _start
             _min = min(list_countdown) - _tot
+            print(f"list_countdown: {list_countdown}")
```
Comment (akasakaid) [inline bot.py:124]: There is no need to display the list_countdown output.

File: bot.py [hunk 2]
```diff
@@ -126,13 +121,14 @@ def main(self):
...
+            print(f"_start: {_start}, _end: {_end}, _tot: {_tot}, _min: {_min}")
             if _min <= 0:
                 continue
 
             self.countdown(_min)
 
-
-if __name__ == "__main__":
+if name == "__main__":
```
Comment (akasakaid) [inline bot.py:131]: name? Isn't it __name__?

Comment (FarmTon) [reply bot.py:131]: __name__ was throwin syntax error for me

File: bot.py (hunk 1 above)
Comment (FarmTon) [reply bot.py:124]: I know its useless i did it for debug purposes

File: bot.py (hunk 2 above)
Comment (akasakaid) [reply bot.py:131]: i mean `__name__`  not only `name` ?

**Comment by akasakaid** [comment]
Alright, I'll do your merge pr. Thank you for contributing to this repository, for my comment above, maybe I will fix it again later.

**Comment by FarmTon** [comment]
You can update it as u wish its more like a first aproach im checking dejentod rn too see if i can fix my problems there.

**Comment by akasakaid** [comment]
i will check dejentod later

**Comment by FarmTon** [comment]
nice , from my tests works clean sending less request with + taps per request (1/2k tested you can maybe go higher)
I personally find 00:01 error after failed to send tap or bot simply gets stuck, tried to apply some checks to see if works atm it does. 
imagen

imagen

**Comment by FarmTon** [comment]
I know failed tap check was added but for some reason wasnt working for me

**Pull request merged**